### PR TITLE
Lifting up again

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
Just passed the control to the game component, not much more. I modified the code in the tutorial so all the control or most of it is on the game component. The Square component is a class again now.

## Screenshot
lifting-up-again

## Trello Card
https://trello.com/b/kJDg7q9d/training-react-rodrigo-hern%C3%A1ndez

### CODE REVIEW

File: rh-tictactoe/src/app/screens/Game/index.js [hunk 1]
```diff
@@ -28,3 +55,23 @@ class Game extends React.Component {
 }
 
 export default Game;
+
+function calculateWinner(squares) {
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/screens/Game/index.js]: extract this function in a `utils` file

File: rh-tictactoe/src/app/screens/Game/index.js [hunk 2]
```diff
@@ -8,15 +8,42 @@ class Game extends React.Component {
...
+  };
+
   render() {
     return (
       <div className={Styles.game}>
         <div className={Styles.gameBoard}>
-          <Board />
+          <Board
+            squares={this.state.history[this.state.history.length - 1].squares}
+            onClick={i => this.handleClick(i)}
```
Comment (guidoprinc) [inline rh-tictactoe/src/app/screens/Game/index.js]: extract this arrow function

File: rh-tictactoe/src/app/screens/Game/index.js (hunk 1 above)
Comment (rodrigoWolox) [reply rh-tictactoe/src/app/screens/Game/index.js]: Done!

File: rh-tictactoe/src/app/screens/Game/index.js (hunk 2 above)
Comment (rodrigoWolox) [reply rh-tictactoe/src/app/screens/Game/index.js]: Done!

**Pull request merged**